### ISSUE TITLE
Incompatible with Lambda Insights Layer?

### ISSUE THREAD

**Issue by huntharo** [awslabs/aws-lambda-web-adapter#179]
I've tried using a Zip Lambda on ARM64, deployed with CDK with:

```
insightsVersion: lambda.LambdaInsightsVersion.VERSION_1_0_135_0,
```

This adds the Insights layer in the Zip Lambda.

I've tried having the Insights layer be merged first or second... doesn't seem to matter.

The Init request fails like this:

<img width="1313" alt="image" src="https://user-images.githubusercontent.com/5617868/221365724-99083cf9-4fdc-44ec-8ede-8212d1c24416.png">

The subsequent requests fail like this:

<img width="1321" alt="image" src="https://user-images.githubusercontent.com/5617868/221365749-78ae1f90-f63f-4159-b87e-fc4488a27206.png">

**Comment by bnusunny** [comment]
This should work. Which port your application is using? I suspect that Lambda Insights Extension also listen on that port. 

I tested it with a FastAPI app listening on port 8000, and it works without any issue.

**Comment by huntharo** [comment]
Ha!  I hadn't even thought that CloudWatch Insights was listening on a port which... what?  Why is it listening on a port?

In any case, I'm using port 3000 so will try swapping that and see if the issue goes away... then I'll want to figure out why they need a port at all.

**Comment by bnusunny** [comment]
CloudWatch Lambda Insight Extension seems to use Log API to collect application logs. It runs a log server on port 3000. Log API will POST log data to that port. 

When I changed the FastAPI app's port to 3000, I got the same error you saw. So here we have the confirmation.

**Comment by huntharo** [comment]
Awesome, thanks, will change the port.

Maybe a small doc update to mention that port 3000 should not be used and why would be sufficient to avoid trouble for others?

**Comment by bnusunny** [comment]
Good idea. Could also cover other not available ports: 1-1024 and 9001.

**Comment by huntharo** [comment]
Confirmed that Insights is working when my app is on port 3001 now.  I'll try to make the docs PR within a few days unless you get to it first.

**Comment by bnusunny** [comment]
I probably could update the doc this weekend.
